Release 1.15.0 (#877)
astronomer/astronomer-providers: CHANGELOG.rst

Changelog
=========

1.15.0 (2023-02-15)
-------------------

New Operators
"""""""""""""

> This release adds the following 1 new async sensor.

Reviewer: ```suggestion
This release adds a new async sensor ``SnowflakeSensorAsync``.
```

Author: Changed

.. list-table::
   :header-rows: 1

   * - Sensor Class
     - Import Path
     - Example DAG

   * - ``SnowflakeSensorAsync``
     - .. code-block:: python

        from astronomer.providers.snowflake.sensors.snowflake import SnowflakeSensorAsync
     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/snowflake/example_dags/example_snowflake_sensor.py>`__

Enhancements
""""""""""""

- Support tags param in ``EmrContainerOperatorAsync``.
  (`#836 <https://github.com/astronomer/astronomer-providers/pull/836>`_)

> - Enhance dbt cloud connection to accept entire tenant domain name.

Reviewer: ```suggestion
- Enhance ``DbtCloudHookAsync`` to accept tenant domain name while making connection.
```

Author: Changed

  (`#855 <https://github.com/astronomer/astronomer-providers/pull/855>`_)

Bug Fixes
"""""""""

> - Make ``DataprocLink`` to be visible for the dataproc jobs in ``DataprocSubmitJobOperatorAsync``.

Reviewer: ```suggestion
- Ensure ``DataprocLink`` is visible for tracking the dataproc jobs status in ``DataprocSubmitJobOperatorAsync``.
```

Author: Changed
